packages/cli/__tests__/zostso/__system__/stop/__scripts__/address-space/as_fully_qualified.sh

```diff
@@ -7,7 +7,8 @@ PORT=$3
 USER=$4
 PASSWORD=$5
 
-SERVLET_KEY=`zowe tso start as --host $HOST --account $ACCOUNT --port $PORT --user $USER --password $PASSWORD --ru=false | grep -oP "(?<=: ).*"`
+# SERVLET_KEY=`zowe tso start as --host $HOST --account $ACCOUNT --port $PORT --user $USER --password $PASSWORD --ru=false | grep -oP "(?<=: ).*"`
```

Comment:
Is there a reason why we are keeping the original statement around as a comment in this file and the next 2 files?